AppCenterCrashes/AppCenterCrashes/WrapperSDKUtilities/MSACWrapperCrashesHelper.h

```diff
@@ -14,35 +14,35 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * This general class allows wrappers to supplement the Crashes SDK with their own behavior.
  */
+NS_SWIFT_NAME(WrapperCrashesHelper)
 @interface MSACWrapperCrashesHelper : NSObject
 
 /**
- * Sets the crash handler setup delegate.
+ * The crash handler setup delegate.
  *
- * @param delegate The delegate to set.
  */
-+ (void)setCrashHandlerSetupDelegate:(id<MSACCrashHandlerSetupDelegate>)delegate;
+@property(class, nonatomic) _Nullable id<MSACCrashHandlerSetupDelegate> crashHandlerSetupDelegate;
 
 /**
  * Gets the crash handler setup delegate.
  *
+ * @deprecated
+ *
  * @return The delegate being used by Crashes.
  */
-+ (id<MSACCrashHandlerSetupDelegate>)getCrashHandlerSetupDelegate;
++ (id<MSACCrashHandlerSetupDelegate>)getCrashHandlerSetupDelegate DEPRECATED_MSG_ATTRIBUTE("Use crashHandlerSetupDelegate instead");
```

Comment:
Any reason to not just remove this API? Isn't it internal? For our wrapper SDKs?

Reply:
This feature was not intended to change Objective-C API, so we decided not to break all wrappers by this.